stock_picking_auto_create_lot_qty/models/stock_lot.py

```diff
@@ -0,0 +1,13 @@
+# Copyright Cetmix OU 2024
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from odoo import fields, models
+
+
+class StockLot(models.Model):
+    _inherit = "stock.lot"
+
+    name = fields.Char(
+        default=lambda self: self.env["ir.sequence"].next_by_code(
+            self.env.company.stock_lot_serial_sequence_id.code
+        )
+    )
```

Comment:
I suggest putting this to create function, as we should only increase LOT sequence when creating new LOT. Putting this to default will increase the sequence even when we are not committing to create new LOT.

Default LOT name can be `_("New")`

Reply:
I agree with you, but I think if we do this, we will break the standard mechanism.
please check [this])(https://github.com/odoo/odoo/blob/4be86b2b30d67c35a3b906b0b94581d1dc159e46/addons/stock/models/stock_lot.py#L29)